crown-court-contribution/src/main/java/uk/gov/justice/laa/crime/contribution/service/ContributionService.java

```diff
@@ -190,11 +187,4 @@ public boolean hasApplicationStatusChanged(RepOrderDTO repOrderDTO, CaseType cas
                 && !repOrderDTO.getRorsStatus().equalsIgnoreCase(status);
     }
 
-    public void requestTransfer(final ApiContributionTransferRequest transferRequest) {
-        UpdateContributionRequest updateRequest = new UpdateContributionRequest()
-                .withId(transferRequest.getContributionId())
-                .withTransferStatus(TransferStatus.REQUESTED)
-                .withUserModified(transferRequest.getUserModified());
-        maatCourtDataService.updateContribution(updateRequest);
```

Comment:
Can we remove the `maatCourtDataService.updateContribution` as well? It looks like with the removal of the above it's now only called by a unit test (which can be removed aswell).

Reply:
Done.